utils/mlflow_tracker.py

```
@@ -101,6 +101,15 @@ def setup(self) -> bool:
             mlflow.set_tracking_uri(self.config.tracking_uri)
             logger.info(f"MLflow tracking URI: {self.config.tracking_uri}")
 
+            # Log the mlflow ui command for convenience
+            # Extract the database path from the URI for the user
+            if self.config.tracking_uri.startswith("sqlite:///"):
+                db_path = self.config.tracking_uri[len("sqlite:///"):]
```

Ruff (0.14.8)

Local variable `db_path` is assigned to but never used

Remove assignment to unused variable `db_path`

(F841)